Funcky.Money.SourceGenerator/Iso4217RecordGenerator.cs

```diff
@@ -40,8 +40,8 @@ private static string GenerateCurrencyClass(IReadOnlyList<Iso4217Record> records
                $"{{{NewLine}" +
                $"{Indent}public partial record Currency{NewLine}" +
                $"{Indent}{{{NewLine}" +
-               $"{GenerateAllCurrenciesProperty(records)}" +
                $"{GenerateCurrencyProperties(records)}" +
+               $"{GenerateAllCurrenciesProperty(records)}" +
```

Comment:
All values in AllCurrencies were null due to the order of static initialisation

Reply:
Silly me 😆